torchaudio/_soundfile_backend.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+import torch
+
+
+def check_input(src):
+    if not torch.is_tensor(src):
+        raise TypeError("Expected a tensor, got %s" % type(src))
+    if src.is_cuda:
+        raise TypeError("Expected a CPU based tensor, got %s" % type(src))
+
+
+def load(
+    filepath,
+    out=None,
+    normalization=True,
+    channels_first=True,
+    num_frames=0,
+    offset=0,
+    filetype=None,
+):
```

Comment:
I notice you're repeating the docstring here. But you call into the load backends separately. I don't think the user will be able to see this unless she looks at this function which is part of a private backend. Here assigning a backend function to the module unction could help use actually choose the correct doc string at runtime.

However, the static documentation won't be able to do that.

So instead I'd say it makes sense to have a single docstring for our load function and then reference it here.

Reply:
When you say "reference", I assume you mean 
```
"""See torchaudio.save"""
```

Unless you meant copying docstring from another with something like `functools.wraps()`?

Reply:
Yes that's correct.